third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -75,6 +75,41 @@ func TestAccBigtableInstance_cluster(t *testing.T) {
 	})
 }
 
+func TestAccBigtableInstance_cluster_resize(t *testing.T) {
+	t.Parallel()
+
+	instanceName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigtableInstanceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigtableInstance_one_in_the_cluster(instanceName, 3),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
After rebasing, you'll probably need to change to use "import-style" checks. See https://github.com/GoogleCloudPlatform/magic-modules/pull/2409/files#diff-1296b3b7cf174ed6b01d0139e46bdd24L69-R76.

If you're seeing differences on `cluster` in "ImportStateVerify" that you don't expect, flag them here and don't worry about fixing them before pushing.